packages/common/workflows.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
+
+const { getS3Object, listS3ObjectsV2 } = require('./aws');
+
+const templateKey = (stack) => `${stack}/workflow_template.json`;
+
+const workflowTemplateUri = (bucket, stack) => `s3://${bucket}/${templateKey(stack)}`;
+
+/**
+ * Get the template JSON from S3 for the workflow
+ *
+ * @param {string} stackName - Cloud formation stack name
+ * @param {string} bucketName - S3 internal bucket name
+ * @returns {Promise.<Object>} template as a JSON object
+ */
+function getWorkflowTemplate(stackName, bucketName) {
+  const key = templateKey(stackName);
+  return getS3Object(bucketName, key)
```

Comment:
I guess I don't feel strongly about whether these should be `async` functions. Thoughts?

Reply:
I don't feel strongly either. My only thought is that we seem to use `await` throughout most of our code now, so it would be nice for consistency. 

This is the only other argument I've found in favor of `await`, but still, I don't feel strongly.

https://mathiasbynens.be/notes/async-stack-traces

Reply:
44974cb9a